app/instance-initializers/global.js

```diff
@@ -0,0 +1,10 @@
+export function initialize(appInstance) {
```

Comment:
We need to either:
1. Communicate this change to the lookit team so they can borrow this implementation
2. Move this to the addon code. Given that this is a hard dependency of the exp-models, I feel it may be better off there. 

Thoughts?